### PR TITLE
Feature - Add a location of a fire with an image

### PR DESCRIPTION
**Pull request by dasunsilva**

### CODE REVIEW

File: components/common/IconButton.tsx
Comment (dasuntheekshanagit) [inline components/common/IconButton.tsx:1]: Cannt we use paper button instead touchable opacity

File: app/WildFireAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+              <Text style={styles.cameraButtonText}>Cancel</Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+      )}
+    </>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (dasuntheekshanagit) [inline app/WildFireAlert.tsx:120]: can we use tailwind css classes

File: components/common/CustomButton.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { GestureResponderEvent, Text, TouchableOpacity } from "react-native";
+
+interface CustomButtonProps {
+  title: string;
+  onPress?: (event: GestureResponderEvent) => void;
+}
+
+const CustomButton = ({ title, onPress }: CustomButtonProps) => {
+  return (
+    <TouchableOpacity
```
Comment (harith-abeysinghe) [inline components/common/CustomButton.tsx:12]: Paper or Touchable Opacity?

**Review by harith-abeysinghe — COMMENTED** [review]
Done

**Review by dasuntheekshanagit — APPROVED** [review]

**Review by harith-abeysinghe — COMMENTED** [review]
Done

**Review by harith-abeysinghe — APPROVED** [review]

**Pull request merged**